docs/regmap/adi_regmap_xcvr.txt

```diff
@@ -110,7 +110,7 @@ ENDFIELD
 REG
 0x0007
 FPGA_INFO
-FPGA device information [[https://github.com/analogdevicesinc/hdl/blob/master/library/scripts/adi_xilinx_device_info_enc.tcl |Xilinx encoded values]]
+FPGA device information [[https://github.com/analogdevicesinc/hdl/blob/main/library/scripts/adi_xilinx_device_info_enc.tcl |Xilinx encoded values]]
```

Comment:
Note to myself: when importing this documentation, update the  regmap link to sphinx style.